src/test/java/org/sagebionetworks/bridge/exporter/handler/SynapseExportHandlerTest.java

```diff
@@ -1,23 +1,13 @@
 package org.sagebionetworks.bridge.exporter.handler;
 
-import static org.mockito.Matchers.any;
-import static org.mockito.Matchers.eq;
-import static org.mockito.Matchers.notNull;
-import static org.mockito.Matchers.same;
-import static org.mockito.Mockito.doReturn;
-import static org.mockito.Mockito.mock;
-import static org.mockito.Mockito.never;
-import static org.mockito.Mockito.spy;
-import static org.mockito.Mockito.verify;
-import static org.mockito.Mockito.when;
-import static org.testng.Assert.assertEquals;
-import static org.testng.Assert.assertNull;
-import static org.testng.Assert.assertTrue;
-import static org.testng.Assert.fail;
+import static org.mockito.Matchers.*;
+import static org.mockito.Mockito.*;
+import static org.testng.Assert.*;
```

Comment:
We usually import these individually. It's trivial, but I wouldn't break with this as the next checkin, there'll be a diff putting them all back again.